addons/mail/tests/test_ir_ui_menu.py

```diff
@@ -0,0 +1,136 @@
+        new_menu('Contacts', parent_id=menu_root_contact.id,
+                 action=new_action('Contacts', res_model='res.partner', view_mode='kanban,tree,form,activity',
+                                   context="{'default_is_company': True}",
+                                   domain='[]'))
+        menu_root_invoicing = new_menu('Invoicing')
+        menu_invoicing_customer = new_menu('Customers', parent_id=menu_root_invoicing.id)
+        new_menu('Customers', parent_id=menu_invoicing_customer.id,
+                 action=new_action('Customers', res_model='res.partner', view_mode='kanban,tree,form',
+                                   context="""{
+                                                'search_default_customer': 1,
+                                                'res_partner_search_mode': 'customer', 
+                                                'default_is_company': True, 
+                                                'default_customer_rank': 1
+                                              }"""))
+        menu_root_sales = new_menu('Sales')
+        menu_sales_orders = new_menu('Orders', parent_id=menu_root_sales.id)
+        new_menu('Customers', parent_id=menu_sales_orders.id,
+                 action=new_action('Customers', res_model='res.partner', view_mode='kanban,tree,form',
+                                   context="""{
+                                                'search_default_customer': 1,
+                                                'res_partner_search_mode': 'customer', 
+                                                'default_is_company': True, 
+                                                'default_customer_rank': 1
+                                              }"""))
+        menu_root_settings = new_menu('Settings')
+        menu_settings_user_and_companies = new_menu('Users & Companies', parent_id=menu_root_settings.id)
+        new_menu('Companies', parent_id=menu_settings_user_and_companies.id,
+                 action=new_action('Companies', res_model='res.company', view_mode='tree,kanban,form'))
+
+        cls.menu_count = new_menu.call_count
+        cls.menu_root_contact = menu_root_contact
+        cls.menu_root_sales = menu_root_sales
+        cls.menu_root_settings = menu_root_settings
+
+        cls.user_public = mail_new_test_user(cls.env, login='user_public', groups='base.group_public',
+                                             name='Bert Tartignole')
+        cls.user_portal = mail_new_test_user(cls.env, login='user_portal', groups='base.group_portal',
+                                             name='Chell Gladys')
+
+    def patch_get_backend_root_menu_ids(self, model, return_values):
+        return patch.object(model.__class__, '_get_backend_root_menu_ids', return_value=return_values)
+
+    def test_initial_data(self):
+        self.assertEqual(len(self.env['ir.ui.menu']._visible_menu_ids()), self.menu_count)
+
+    @warmup
+    @users('__system__', 'employee')
```

Comment:
I think we can keep employee only, using system in mail tests is just coming from old tests when admin was system